modules/mo_pytorch/mo_extensions/front/pytorch/pooling_ext.py

```diff
@@ -78,3 +78,39 @@ def extract(cls, node):
         }
         AdaptiveAvgPooling.update_node_stat(node, data)
         return cls.enabled
+
+
+class AvgPool2dFrontExtractor(FrontExtractorOp):
+    op = 'AvgPool2d'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        # Extract pads attribute
+        pads = np.array([node.module.padding, node.module.padding], dtype=np.int64).reshape(1, 2)
+        pads = np.repeat(pads, 2, axis=0)
```

Comment:
I think that we need to have a unified approach for paddings across layers. Should we do the same as proposed for conv layer?